CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Swift Next
   }
   ```
 
+* [#6144]
+
+  Remove the `system-module` and `manifest` templates and clean up the remaining `empty`, `library`, and `executable` templates so they include the minimum information needed to get started, with links to documentation in the generated library, executable, and test content.
+
```

Comment:
Since this isn't guarded behind anything, I think you can already put this in the 5.9 section instead. We use this section for features that don't have a clear target yet and are guarded behind tools-version 999.0 etc

Reply:
Sounds good, I moved it down.